tracer.go

```diff
@@ -210,3 +210,16 @@ func (t *tracerImpl) Join(operationName string, format interface{}, carrier inte
 	}
 	return nil, opentracing.ErrUnsupportedFormat
 }
+
+// GetGlobalTracerOptions attempts to get the basictracer.Options
+// struct provided to the global Tracer.  Returns the options and true
+// if the global Tracer is a basictracer implementation; returns
+// false otherwise.
+func GetGlobalTracerOptions() (Options, bool) {
```

Comment:
Ultimately, we are trying to get from the result of opentracing.GetGlobalTracer() to the implementation of basictracer.Recorder. Exposing the Options seems useful and gives me a pointer to the Recorder. What would you suggest?

Following your earlier comment, I could make an interface to obtain the Options or the Recoder from a basictracer-supplied Tracer.